cavern/src/main/java/org/opencadc/cavern/files/FileAction.java

```diff
@@ -83,22 +84,22 @@
 import org.opencadc.vospace.server.LocalServiceURI;
 import org.opencadc.vospace.server.NodePersistence;
 import org.opencadc.vospace.server.PathResolver;
+import org.opencadc.vospace.server.actions.BaseFileAction;
+import org.opencadc.vospace.server.actions.NodeAction;
 import org.opencadc.vospace.server.auth.VOSpaceAuthorizer;
 
 /**
  *
  * @author majorb
  * @author jeevesh
  */
-public abstract class FileAction extends RestAction {
+public abstract class FileAction extends BaseFileAction {
```

Comment:
revert most code changes in this cavern package; just keep the minimum required by API change in TransferGenerator

Reply:
Done